src/posthog-surveys.ts

```diff
@@ -265,8 +265,9 @@ export class PostHogSurveys {
 
     // this method is lazily loaded onto the window to avoid loading preact and other dependencies if surveys is not enabled
     private _canActivateRepeatedly(survey: Survey) {
-        if (isNullish(assignableWindow.__PosthogExtensions__.canActivateRepeatedly)) {
+        if (isNullish(assignableWindow.__PosthogExtensions__?.canActivateRepeatedly)) {
             logger.warn(LOGGER_PREFIX, 'canActivateRepeatedly is not defined, must init before calling')
+            return false // TODO does it make sense to have a default here?
```

Comment:
@jurajmajerik @dmarticus i'm unfamiliar with this (and lazy 🤣 )

is it safe to return false here? will we accidentally disable surveys or something?

Reply:
I think this is fine.

This method is called here: https://github.com/PostHog/posthog-js/blob/fix/warn-not-throw/src/posthog-surveys.ts#L182

If you don't return false early than an error will be thrown on return.

@Phanatic can you double check please? I believe you've added this recently.